### DISCUSSION TITLE
Zephyr BLE OS Discoverability and Pairing Issues (Windows, Mac, iOS, Android)

### DISCUSSION THREAD

**Discussion by nonlin** [zephyrproject-rtos/zephyr#73080] [Q&A]
I've gotten some success using Zephyr for enabling BLE on my hardware but I'm running into some odd issues that I'm hoping might be common knowledge. 

My hardware is scannable and connectable if I have my app doing a BLE scan. But when it comes to discoverability and connecting via an OS Scan I seem to run into issues. 

Mac/iOS can't ever find my hardware via OS BLE Discovering. 
Android lists it while searching for BLE Devices but can't pair. 
Windows under advanced view will also show my device but has issues connecting as well. Windows seems to auto disconnect giving reason 0x13 on device. 

Is there a sample project that might help illuminate how to solve that? Or is there any known common mistakes/issues that might cause that to happen? Maybe its a config issue?

**Comment by RomainPelletant** [comment]
Can you share with us the hardware used, Zephyr revision used and potentially some logs?
